arch/x86/kvm/pkvm/cpu.c

```diff
@@ -22,6 +22,8 @@ unsigned int nr_cpu_ids;
 DEFINE_PER_CPU(u64, x86_spec_ctrl_current);
 DEFINE_STATIC_KEY_FALSE(switch_vcpu_ibpb);
 unsigned int tsc_khz;
+DEFINE_PER_CPU(struct task_struct, pkvm_task);
```

Comment:
nit: maybe let's add a comment like "Used for switching the FPU state between the host and pVMs"? (Since otherwise it is quite non-obvious why pKVM needs to do anything with task structs, given that pKVM doesn't do any scheduling.)